### ISSUE TITLE
UML class diagram doesn't "see" classes in namespaces

### ISSUE THREAD

**Issue by selganor74** [alm-tools/alm#105]
I'm trying to get a class diagram of my project, which is still structured as a Global namespace (I know, I should move to external modules, but I have a lot of preexisting code that it is still not ready! :smile: ) ,

My code looks like this:

```
namespace DDDTools.Repository {

[...]

    export abstract class BaseRepositoryAsync<T extends BaseAggregateRoot<T, TKey>, TKey extends IKeyValueObject<TKey>>
        implements IRepositoryAsync<T, TKey> {

[...]
```

but when running UML class diagram from alm, I sadly get "No classes in file"... Am I missing something ?

P.S.: Thank you for alm, it is already a great tool !!!

**Comment by basarat** [comment]
I was just lazy and ignored this. I'll fix it asap 🌹

**Comment by basarat** [comment]
Released with `0.0.67` :rose:
